chain/network/src/types.rs

```diff
@@ -95,6 +101,9 @@ pub struct KnownPeerState {
     pub status: KnownPeerStatus,
     pub first_seen: time::Utc,
     pub last_seen: time::Utc,
+    // Last time we tried to connect to this peer.
+    // This data is not persisted in storage.
```

Comment:
if you don't want to persist this field, then Option<(time::Utc, anyhow::Result)> would be better.